src/core/CMakeLists.txt

```diff
@@ -1,149 +1,148 @@
 set(EspressoCore_SRC
-  accumulators.cpp
```

Comment:
Is there any specific reason for the indentation change? I feel this will just lead to merge conflicts for no good reason.

Reply:
besides beautification, no

Reply:
but I think we should think about using ```cmake-format```

Reply:
if can revert the formatting changes if needed

Reply:
If a single correct formatting can be enforced by a tool that would be perfect. I think I looked at `cmake-format` once, but the results with the default config were not so good...

Reply:
that should be possible. It probably has the same issues as ```autopep8```, *i.e.* the exact formatting depends on the version of the tool